Playwright-POM-PoC-ts/playwright.config.ts

```diff
@@ -0,0 +1,64 @@
+import { PlaywrightTestConfig, devices } from "@playwright/test";
+import dotenv from "dotenv";
+import path from "path";
+
+/**
+ * Read environment variables from file.
+ */
+dotenv.config({ path: path.resolve(__dirname, ".env"), override: true });
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+  testDir: "./tests",
+  globalSetup: "./utils/global-setup.e2e.ts",
```

Comment:
```suggestion
  globalSetup: "./utils/global-setup.ts",
```

Reply:
In this particular (basic, simple) test U don't need `globalSetup`, but this structure can stay for further use